docs/publishing.md

```diff
@@ -0,0 +1,58 @@
+## Versioning
```

Comment:
let's rename this file to `docs/PUBLISHING.md`

Reply:
🤦  I swore I did that in the other pr but a wrinkle in my brain must have smoothened itself, will do